sklearn/metrics/_plot/tests/test_plot_confusion_matrix.py

```diff
@@ -264,3 +263,35 @@ def test_confusion_matrix_text_format(pyplot, data, y_pred, n_classes,
     text_text = np.array([
         t.get_text() for t in disp.text_.ravel()])
     assert_array_equal(expected_text, text_text)
+
+
+@pytest.mark.parametrize("values_format", [None])
+def test_confusion_matrix_standard_format(pyplot, data, y_pred, n_classes,
+                                          fitted_clf, values_format):
+    X, y = data
+    cm = np.array([[10000000, 0], [29, 123123]])
+    disp = plot_confusion_matrix(fitted_clf, X, y,
+                                 include_values=True,
+                                 values_format=values_format)
+    disp.confusion_matrix = cm
```

Comment:
This approach to modifying the attribute is a pretty strange usage. We don't do it elsewhere in tests or examples. Why not just use `ConfusionMatrixDisplay(cm, [False, True]).plot(values_format=values_format)`?

Reply:
Yea the code becomes much simpler using this line, sorry about that. I'm still new to contributing to projects, so sorry it takes some extra time :).

I just took this testing as it was how I did it in jupy notebooks.